### DISCUSSION TITLE
Skin makes Osu Lazer stuttery

### DISCUSSION THREAD

**Discussion by ManuMTS** [ppy/osu#18588] [Q&A] [answered]
Hello People.
This Skin made Lazer run like ****. It runs perfectly fine in regular osu. It was suggested on ppy's stream that i post it here. So i did.
What Setup am i running? 
- OS: Windows 10 21H2
- CPU: Ryzen 7 3700X
- GPU: RX6900XT

After the first advice i removed all the 1x1 textures of the skin aswell as the animation. The stuttering was still happening though.

Here is the skin: https://gofile.io/d/M0VFbi

**Comment by peppy** [comment, marked as answer]
I.... don't know where to start with this skin. What I can say is that it is *highly unoptimised*, to the point I'm not sure we should be working to support such a creation.

For instance, the `ranking-panel` sprite is 27mb. `section-pass` and `section-fail` are full screen images rather than small glyphs. The skip button is a fullscreen image. There are zero byte `ogg` files (probably not handled correctly). The `skin.ini` contains tab characters before the keys (not sure this is handled correctly).

I don't know which of the above is the issue, but a good starting point would be to try removing one at a time to see what helps.

**Reply by ManuMTS** [reply]
Thanks for looking into the skin. I will forward the issues to Nikstrim the skin creator. I also have the issue on other skins.
For example this whitecat skin. There is also some stuttering when using it in lazer.
[-         《CK》 WhiteCat 2.0 ~ new.zip](https://github.com/ppy/osu/files/8844024/-.CK.WhiteCat.2.0.new.zip)

**Reply by Walavouchey** [reply]
note that for silent hitsounds and other sound effects, you're supposed to use [a specific 44-byte .wav file](https://up.ppy.sh/files/blank.wav)

from the [ranking criteria](https://osu.ppy.sh/wiki/en/Ranking_Criteria#rules.4):

![image](https://user-images.githubusercontent.com/36758269/172174239-d2747b8f-05df-42e5-a9c3-3ba538dbf68e.png)
